Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.yml

```diff
@@ -106,6 +106,7 @@ configuration:
   section: Collect
   advanced: true
   required: false
+  additionalinfo: If this option is enabled and you're fetching emails by modification time, it could lead to duplicate entries. This is because marking an email as read changes its metadata, causing it to be re-fetched in the next cycle.
```

Comment:
I'm not sure we would like to publish a note as choosing this option could lead to a bug.... If there is a bug in the process we should fix it